pages/code-review.md

```diff
@@ -1,6 +1,143 @@
+- Identify ways to simplify the code while still solving the problem.
+- Offer alternative implementations, but assume the author already considered
+  them. ("What do you think about such-and-such here?")
+- If discussions turn too philosophical or academic, move the discussion to a new
+  issue or offline. In the meantime, the author has the final say on the current
+  implementation.
+- Sign off on the pull request with a :thumbsup: or "Ready to merge" comment.
+- Wait to merge the branch until it has passed Continuous Integration testing.
+  (TDDium, TravisCI, etc.)
+- Merge once you feel confident in the code and its impact on the project.
+
+
+## <a name="submitting"></a>Submitting code
+
+Before seeking a review, you should be able to check off each of the following:
+
+    - [ ] Changes are limited to a single goal (no scope creep)
+    - [ ] Code can be automatically merged (no conflicts)
+    - [ ] Code follows the standards laid out in this playbook
+    - [ ] Passes all existing automated tests
+    - [ ] New functions include new tests
+    - [ ] New functions are documented (with a description, list of inputs, and
+         expected output)
+    - [ ] Placeholder code is flagged
+    - [ ] Visually tested in supported browsers and devices
+    - [ ] Project documentation has been updated (including the "Unreleased" section of
+          the CHANGELOG)
+
+### Opening a PR
+
+Once sure your code meets the checks above, open a Pull Request for your new
+feature or bugfix following CFPB's [PR Template](https://raw.githubusercontent.com/cfpb/front-end/master/pr-template.md). You can also create a [Bookmarklet based on this template](https://gist.github.com/cfarm/b9b638943a2eea52a3a8). Remember that the
+more information you provide to reviewers, the more context they will have. This
+leads to faster reviews, and less back and forth between everyone.
+
+
+## Reviewing code
+
+When reviewing code, you should be able to check off each of the following:
+
+    - [ ] Do the changes address the project's needs?
+    - [ ] Do the changes respect the project's existing style?
+    - [ ] Does the new code avoid reproducing existing functionality?
+    - [ ] Are functions as simple as possible?
+    - [ ] Is the code as efficient as possible?
+    - [ ] Is the usage of each function clear?
+    - [ ] Have edge cases been considered and tested for?
+
+Use this bookmarklet code to add this checklist to your Pull Request comments: [Code Review Checklist](https://gist.github.com/cfarm/a4174fe6f775353a3115)
+
+### Commenting on code
+
+When commenting on breaks a team's style guide, reviewers should include a link back
+to the appropriate guide.
+
+Ex:
+
+    [CSS - Property Values](https://github.com/cfpb/front-end/css.md#property-values):
+    If the value of a property is `0`, do not specify units
+
+If you disagree with a guideline or there is no current guideline, open an
```

Comment:
I'll remove the bits about style guides throughout; I don't believe they're really fleshed out anywhere